linter/src/lint/lint.cpp

```diff
@@ -1,58 +1,57 @@
 #include "lint.hpp"
 
+#include "pywrapper/runtime.hpp"
+
+#include <fmt/core.h>
+#include <pybind11/pybind11.h>
+
+#include <optional>
+#include <string>
+
 namespace nutc {
 namespace lint {
-bool
-lint(
-    const std::string& uid, const std::string& algo_id, std::stringstream& output_stream
-)
+
+static bool
+mock_market_func(const std::string&, const std::string&, float, float)
 {
-    std::optional<std::string> algoCode = nutc::client::get_algo(uid, algo_id);
-    if (!algoCode.has_value()) {
-        output_stream << "[linter] FAILURE - could not find algo id " << algo_id
-                      << " for uid " << uid << "\n";
-        nutc::client::set_lint_failure(uid, algo_id, "Could not find algo id");
-        return false;
-    }
+    return true;
+}
 
-    bool e = nutc::pywrapper::create_api_module(nutc::mock_api::getMarketFunc());
-    if (!e) {
-        log_e(linting, "Failed to create API module");
-        output_stream << "[linter] failed to create API module\n";
-        nutc::client::set_lint_failure(uid, algo_id, "Failed to create API module");
-        return false;
+lint_result
+lint(const std::string& algo_code)
+{
+    std::string out_message = "[linter] starting to lint algorithm\n";
+    bool ok = nutc::pywrapper::create_api_module(mock_market_func);
+    if (!ok) {
+        out_message += "[linter] failed to create API module\n";
+        return {false, out_message};
+    }
+    ok = nutc::pywrapper::supress_stdout();
```

Comment:
I'm not sure if we actually want to supress stdout in the linter - people may use it for debug